packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -191,6 +195,12 @@ marketVolConfigurator18 = "<%= settings.storkExecutor18 %>"
 marketVolConfigurator19 = "<%= settings.storkExecutor19 %>"
 marketVolConfigurator20 = "<%= settings.storkExecutor20 %>"
 
+# TODO: add proper addresses
```

Comment:
on testnet it's potentially fine to keep these addresses

Reply:
yes